census/estimator/trainer/model.py

```diff
@@ -236,7 +234,7 @@ def json_serving_input_fn():
       key: tf.expand_dims(tensor, -1)
```

Comment:
Supposedly, this is no longer needed. So you should be able to just return

return tf.estimator.export.ServingInputReceiver(inputs, inputs)

Likely applies to example_serving_input_fn